### PR TITLE
TASK-21: Show events as a list front end

### LINKED ISSUE #76: TASK-21: Show Events as a list Front-end
**Task Description**

As a developer I want to be able to display a list of events with the event information on the discover page in the Discover/Upcoming/My Events tabs.

**Subtasks**

**Acceptance Criteria**

- [ ] Documentation present where necessary
- [ ] Unit and UI tests are implemented for every component and function
- [ ] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2000%3A343283)

### PR DESCRIPTION
**Pull request by r-saba**
closes #76 - The discover page now shows a list of events. 

**Notes**
The list of events is stored in a variable for this PR. It will come from a graphql query in #65

### CODE REVIEW

**Review by leojrsilao — DISMISSED** [review]
Everything LGTM! Good work 🚀🚀

- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every component and function
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2000%3A343283)

**Review by ys-lin — DISMISSED** [review]
Looks good to me 👍 
- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every component and function
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2000%3A343283)

File: apps/mull-ui/src/app/components/event-card/event-card.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 .event-card-container {
   position: relative;
   box-shadow: 0 0.25rem 0.25rem rgba(0, 0, 0, 0.25);
+  margin-bottom: 2rem;
```
Comment (cristian-aldea) [inline apps/mull-ui/src/app/components/event-card/event-card.scss]: I don't think this margin should always be there. Whatever page is using the event card can decide how spaced out it is from other components.

It doesn't make much of a difference, but I think the component should decide how it looks and is formatted, but shouldn't worry about its positioning or spacing with other components, except in special cases like the navigation components.

**Review by cristian-aldea — DISMISSED** [review]
- [X] Documentation present where necessary
- [X] Unit and UI tests are implemented for every component and function
- [X] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2000%3A343283)

Just had a small comment, but otherwise LGTM

**Review by JimLau49 — DISMISSED** [review]
Good work Ragith !
- [x] Documentation present where necessary
- [x] Unit and UI tests are implemented for every component and function
- [x] Must match [Mockup](https://www.figma.com/file/FJPXMYI4ULCnuCEZvPs67k/Mull?node-id=2000%3A343283)

File: apps/mull-ui/src/app/components/event-card/event-card.scss (hunk 1 above)
Comment (r-saba) [reply apps/mull-ui/src/app/components/event-card/event-card.scss]: Good point. Didn't notice it's used in other places. with different spacing.

I moved this into the `discover.scss`. The inline style was introducing a weird workflow where the component was handling some styling and the scss was handling other parts of the styling and was forced to use `!important` to overwrite the inline style.

We can still use the `style` prop on event card for cases like if we want to be display none when some variable is false.
Let me know what you think.

**Review by JimLau49 — DISMISSED** [review]

File: apps/mull-ui/src/app/components/event-card/event-card.scss (hunk 1 above)
Comment (cristian-aldea) [reply apps/mull-ui/src/app/components/event-card/event-card.scss]: yeah sorry about that, the style thing was probably a remnant of me setting it up as a placeholder demo

**Review by cristian-aldea — DISMISSED** [review]

**Review by JimLau49 — APPROVED** [review]

**Review by leojrsilao — APPROVED** [review]

**Review by cristian-aldea — APPROVED** [review]

**Pull request merged**